src/ducks/search/SearchPage.jsx

```diff
@@ -44,7 +44,9 @@ const SearchPage = () => {
   const { t } = useI18n()
   const { isMobile } = useBreakpoints()
 
-  const [search, setSearch] = useState(params.search || '')
+  const [search, setSearch] = useState(
+    params.search ? decodeURIComponent(params.search) : ''
```

Comment:
Can you add a simple test plz ?

Reply:
What do you want me to test?

Reply:
By checking that the content is correctly put in the href (${encodeURIComponent(label)}) and correctly displayed in the search page?

Reply:
So testing encodeURIComponent and decodeURIComponent? And testing react-router? I'll not do that ^^' 

There is already a test to check that what is in the URL in used to make the search. I personally think that it's enough

Reply:
It wasn't really the idea. But rather to check that if a transaction's label has a "/" (for example), so it must be encoded and not displayed in raw in order to avoid this kind of situation.